tests/network/primary_pod_network.go

```diff
@@ -178,13 +178,30 @@ func vmiWithMasqueradeBinding() *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func newFedoraWithGuestAgent() (*v1.VirtualMachineInstance, error) {
+func newFedoraWithGuestAgentAndMasqueradeBinding() (*v1.VirtualMachineInstance, error) {
 	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
 	if err != nil {
 		return nil, err
 	}
 
 	tmpVmi := libvmi.NewFedora(
+		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
```

Comment:
Can you pass `libvmi.InterfaceDeviceWithMasqueradeBinding()` in as an argument?

Reply:
Done